composer.json

```diff
@@ -25,7 +25,9 @@
         }
     },
     "require": {
-        "omnipay/common": "~2.0"
+        "omnipay/common": "~2.0",
+        "vimeo/payment-gateway-logger": "^1.0",
+        "ext-simplexml": "*"
```

Comment:
Please see https://github.com/vimeo/omnipay-vindicia/pull/57#discussion_r277449134 about why the`ext-*` package was added. A similar thing was done for vindicia except for the soap extension.